README.md

```diff
@@ -72,29 +78,41 @@ curl -s -G --data-urlencode "q=title:list" localhost:9200/try_cloaked_search/_se
 These are a couple examples of simple term queries:
 
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND title:Japan" localhost:8675/try_cloaked_search/_search | jq
+curl -s -G --data-urlencode "q=+tenant_id:tenant-1 AND title:Japan" localhost:8675/try_cloaked_search/_search | jq
 ```
 
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND title:cup" localhost:8675/try_cloaked_search/_search | jq
+curl -s -G --data-urlencode "q=+tenant_id:tenant-1 AND title:cup" localhost:8675/try_cloaked_search/_search | jq
+```
+
+Compare these results to the ones returned by querying Elasticsearch directly:
+```bash
+curl -s -G --data-urlencode "q=+tenant_id:tenant-1 AND title:Japan" localhost:9200/try_cloaked_search/_search | jq
+curl -s -G --data-urlencode "q=+tenant_id:tenant-1 AND title:cup" localhost:9200/try_cloaked_search/_search | jq
+```
+
+Now try querying on a protected field:
+
+```bash
+curl -s -G --data-urlencode "q=+tenant_id:tenant-1 AND summary:glasgow" localhost:8675/try_cloaked_search/_search | jq
 ```
 
 Term queries can also be combined with ORs or ANDs like this:
 
 ```bash
-curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND (title:cup OR title:Japan)" localhost:8675/try_cloaked_search/_search | jq
+curl -s -G --data-urlencode "q=+tenant_id:tenant-1 AND (summary:cup OR title:Japan)" localhost:8675/try_cloaked_search/_search | jq
```

Comment:
This used to show 2 fields that are protected both being queried and now it's one protected and one plaintext. Maybe we should have both types of queries to show that they can be mixed and matched.